romancal/datamodels/schemas/reference_files/referencefile.schema.yaml

```diff
@@ -12,7 +12,8 @@ allOf:
         reftype:
           title: Reference file type
           type: string
-          enum: [FLAT, BASE]
+          enum: [AREA, BASE, DARK, FLAT, GAIN, IPC, LINEARITY, PERSAT, PHOTOM, READNOISE, REFPIX,
+                 SATURATION, SUPERBIAS, TRAPDENSITY, TRAPPARS,]
```

Comment:
I don't think BASE is a valid ref type. That was just my shorthand for a placeholder. 
Should there be a comma at the end ? TRPPARS,]  --> TRAPPARS]

Reply:
Base was a hold over from JWST that i left in case it had some special meaning in the pipeline. I can remove it, if you like.

Python allows for trailing commas in lists (which makes things like copy & paste and appending easier), but I can remove it if you like.